### DISCUSSION TITLE
Return path overriding FROM ADDRESS

### DISCUSSION THREAD

**Discussion by freelovehere** [postalserver/postal#2960] [Help with using Postal]
I would love all the help I can get please, about 3- 4 weeks ago, something changed in my server, I have no idea if this happened due to the update that was made or something but my return path email has been overriding my FROM address. If I send an email, the return path email shows up as From instead of the FROM address thereby making it hard for me hide my FROM address and it has just complicated things.
      I tried to downgrade the postal to an older version and it is still the same issue. I would love all the help I can get and I have attached a sample of the problem. Thank you 
![IMG_1362](https://github.com/postalserver/postal/assets/169105514/259479b3-073c-4d14-8fe0-0d86783e2ecd)

**Comment by willpower232** [comment]
Its more likely that whatever system converts your emails to SMS was changed so you will have to get in touch with them, you would have been aware of any changes to your Postal server.

**Reply by freelovehere** [reply]
Thank you so much, that was my initial thought as well until I used someone else’s postal account, KINGMAILER.org to send emails and behold, the FROM address worked and I also tried sending from a webmail and even from my personal email address and they worked too which is why I am not confused as to what the reason could be
